integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -119,11 +119,6 @@ func (s *DockerExternalVolumeSuite) SetUpSuite(c *check.C) {
 			http.Error(w, err.Error(), 500)
 		}
 
-		p := hostVolumePath(pr.name)
-		if err := os.RemoveAll(p); err != nil {
-			http.Error(w, err.Error(), 500)
-		}
-
 		w.Header().Set("Content-Type", "application/vnd.docker.plugins.v1+json")
```

Comment:
Actually we've not really versioned the volume plugin api.
The generic plugin/framework API is versioned (and hasn't itself changed)